sites/skeleton.dev/src/routes/(inner)/actions/filters/+page.svelte

```diff
@@ -127,7 +127,7 @@ only utilize theme on this doc page.
 				<svelte:fragment slot="panel">
 					{#if method === 0}
 						<p>
-							Use the following <a class="anchor" href="https://svelte.dev/tutorial/actions" target="_blank" rel="noreferrer"
+							Use the following <a class="anchor" href="https://v4.svelte.dev/tutorial/actions" target="_blank" rel="noreferrer"
```

Comment:
@nullpointerexceptionkek I see what you were going for with the subdomain but there's two notable issues:

1. The newest release is Skeleton v3 (not v4)
2. We likely won't generate a subdomain for this until this version is archived.

So the correct links would be, whatever the `next.` URL is now, but dropping the subdomain and using `skeleton.dev`. Like so:

Now:

```
https://next.skeleton.dev/docs/tailwind/buttons
```

After v3 Launches:

```
https://skeleton.dev/docs/tailwind/buttons
```

If you could please submit a revision for all updated URL like this please, then I'll sit on this until just before launch next week.